ci-operator/config/openshift/cluster-api-actuator-pkg/openshift-cluster-api-actuator-pkg-release-4.20__images.yaml

```diff
@@ -0,0 +1,28 @@
+base_images:
+  ocp_builder_rhel-9-golang-1.23-openshift-4.19:
```

Comment:
Do we want to be using 4.20 base images? Do they exist yet?

Reply:
they doesn't exist yet, I can't find the image registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.23-openshift-4.20 anywhere in this repo

Reply:
hmm, I see `ocp_builder_rhel-9-golang-1.23-openshift-4.20`  `rhel-9-golang-1.23-openshift-4.20` being used 

e.g https://github.com/openshift/release/blob/2d7bf0ecc0f8a1ebe03deff257e14a8f9ccdbe18/ci-operator/config/openshift/bond-cni/openshift-bond-cni-main__okd-scos.yaml#L6C3-L9 and https://github.com/search?q=repo%3Aopenshift%2Frelease+rhel-9-golang-1.23-openshift-4.20&type=code ?

Reply:
Yes, thank you, I checked the image exist. Have changed to 4.20.
```
$ oc image info registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.23-openshift-4.20 
  OS            DIGEST
  linux/amd64   sha256:42fe8b20daf4fbf5b7a54c6cae002bf91f379814dfffec8971a7eeaf51a6a6ca
  linux/arm64   sha256:4862e8b7a2d512ff8631a123e97d1a9388f9d71105c2e2355bef87db111edfb5
  linux/ppc64le sha256:b57e6b075ead855e0d26a042e3b080c2f4de3f73a46a0cf6b7364f7a82dc8d6a
  linux/s390x   sha256:be33f9576f7d294333a51f114b78ab500ef16ebfa60ddc0002db9f90cce51574
```